catalog/monitors/k8s.yaml

```diff
@@ -626,3 +626,30 @@ k8s-pending-pods:
     #critical_recovery:
     #warning_recovery:
 
+k8s-metrics-server-low-proc-count:
```

Comment:
We should definitely have some inline documentation on top of this monitor explaining its purpose, as this monitor is a bit unusual and very specific in nature.